src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs

```diff
@@ -68,16 +68,17 @@ public override Task<bool> CanLaunchAsync(DebugLaunchOptions launchOptions)
         /// <summary>
         /// Helper returns the correct debugger engine based on the targeted framework
         /// </summary>
-        public static Guid GetManagedDebugEngineForFramework(string targetFramework)
+        public static Guid GetManagedDebugEngineForFramework(string targetFramework, bool mixedMode)
         {
             // The engine depends on the framework
             if (IsDotNetCoreFramework(targetFramework))
             {
+                if (mixedMode) throw new Exception(VSResources.UnmanagedDebuggingNetCoreNotSupported);
```

Comment:
It is supported in .NET Core 2.0. To enable it - pass ManagedCoreEngine as the launching engine, and NativeOnly in the additional engines. However, there is a bug in VS 2017 15.3 where it doesn't work very well. We can port it to 15.4 if you are trying to enable this scenarios for an earlier VS.